main.py

```diff
@@ -24,8 +25,14 @@ def imgcaption(sid, data):
 
 
 @sio.event
-def chat(sid, data):
-    msg = send_message(send_message(data["message"]))
+def chat(sid, data: dict):
+    if data.get("textgenwui") is not None:
```

Comment:
did you test this? idk how the client side would work

Reply:
i am incapable of testing the main.py parts